src/server-code/main-server.js

```diff
@@ -160,36 +159,10 @@ export default function runMainServer(serverFalcorModel) {
     );
   });
 
+
   if (isStaging) {
-    app.get('/login', (req, res) => {
-      match({ routes, location: req.url },
-        (error, redirectLocation, renderProps) => {
-          if (error) {
-            res.status(500).send(error.message);
-          } else if (redirectLocation) {
-            res.redirect(302, redirectLocation.pathname + redirectLocation.search);
-          } else if (renderProps) {
-            renderApp(renderProps, true).then((html) => {
-              res.status(200).send(html);
-            }).catch((err) => {
-              if (process.env.NODE_ENV !== 'production') {
-                console.error('Failed to render: ', req.url); // eslint-disable-line no-console
-                console.error(err.stack || err); // eslint-disable-line no-console
-                res.status(500).send(err.stack || err);
-              } else {
-                res.status(500).send('There was an error while serving you this webpage.' +
-                  ' Please contact The Gazelle team and tell them this link is broken. We hope' +
-                  ' to fix it soon. Thank you.');
-              }
-            });
-          } else {
-            res.status(404).send('Not Found');
-          }
-        }
-      );
-    });
-    app.get(/(?!\/login)/, (req, res) => {
-      res.redirect(307, `/login?url=${req.url}`);
+    app.get('/robots.txt', (req, res) => {
+      res.status(200).type('txt').send(nothingAllowedRobotsTxt);
     });
   } else {
```

Comment:
If you don't remove this else block, the staging site will now no longer take any calls other than the one for `robots.txt` ;). Just put the `robots.txt` in the if block, and place the main wildcard route afterwards without any conditionals, that one should always be added.